packages/flutter/test/widgets/form_test.dart

```diff
@@ -1457,4 +1457,45 @@ void main() {
 
     expect(focusNode2.hasFocus, isTrue);
   });
+
+  testWidgets('AutovalidateMode.always should validate on first build', (WidgetTester tester) async {
+    String? errorText(String? value) => '$value/error';
```

Comment:
Making the return type non-nullable would remove the need for null checks further down!

```suggestion
    String errorText(String? value) => '$value/error';
```